webui/src/app/stores/badges-store.service.ts

```diff
@@ -3,7 +3,8 @@ import { createFeatureSelector, createSelector, Selector, Store } from '@ngrx/st
 
 import { ActionWithPayloadAndPromise } from './action-with-payload';
 import {  BadgeBean, BadgePagesBean } from '../models/commons/badges-bean';
-import { Subject } from 'rxjs/Subject';
+import { Subject } from 'rxjs';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?